src/towncrier/test/test_build.py

```diff
@@ -1769,3 +1769,54 @@ def test_no_ignore_configured(self, runner):
             _main, ["--draft", "--date", "01-01-2001", "--version", "1.0.0"]
         )
         self.assertEqual(0, result.exit_code, result.output)
+
+    @with_project(
+        config="""
+    [tool.towncrier]
+    title_format = "{version} - {project_date}"
+
+      [[tool.towncrier.type]]
+      directory = "feature"
+      name = "Feature"
```

Comment:
Very minor comment to be explicit about the target of this test

```suggestion
      name = "Feature"
      # showcontent is not defined in TOML
```